definition/data/sheets/AuthorisationCaseEvent.json

```diff
@@ -54,7 +54,7 @@
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "RejectOrganisationPaymentPlan", "UserRole": "citizen", "CRUD": "CR"},
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "AgreementRejectedByDefendant", "UserRole": "citizen", "CRUD": "CR"},
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "AgreementCounterSignedByDefendant", "UserRole": "citizen", "CRUD": "CR"},
-  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "AgreementSignedByBoth", "UserRole": "citizen", "CRUD": "CR"},
```

Comment:
this should be CR

Reply:
For production? Or only when CTSC goes live?

Reply:
This is what is going to production next Monday. For citizen role it is generally safe to leave it as {{CR}}. Agreed?

Reply:
Agreed

Reply:
Code review addressed.